packages/web-app-files/src/views/Personal.vue

```diff
@@ -26,7 +26,9 @@
         :resources="activeFiles"
         :target-route="targetRoute"
         :header-position="headerPosition"
-        @showDetails="$_mountSideBar_showDefaultPanel"
```

Comment:
Current master uses `$_mountSideBar_showDefaultPanel`, could it be your `$_mountSideBar_showDetails` is from an older version of the repo?

Reply:
yes